### PR TITLE
Fix "Manage Organization Settings" article

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) fixes an error I introduced with PR #3086 in this article:
https://help.sumologic.com/docs/manage/manage-subscription/manage-org-settings/

I needed to flip the "Cloud Flex Credits" and "Cloud Flex" instructions. The kebob menu is used for Cloud Flex Credits! 🤦 

Issue number: Original issue was #3084 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, I really goofed on this one.

File: docs/manage/manage-subscription/manage-org-settings.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ For Cloud Flex:
 
 1. In the left navigation bar of the UI, select **Administration** > **Account**. 
 1. Access the appropriate menu for your [account type](#availability):
-     * Cloud Flex Credits: At the top select **Manage Organization > Change Organization Name**.
-     * Cloud Flex: From the kebob menu at the top select **Change Organization Name**.
+     * Cloud Flex Credits: From the kebob menu at the top select **Change Organization Name**.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/manage-org-settings.md:34]:
```suggestion
     * Cloud Flex Credits: From the kebab menu at the top, select **Change Organization Name**.
```

File: docs/manage/manage-subscription/manage-org-settings.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ For Cloud Flex:
...
+     * Cloud Flex: At the top select **Manage Organization > Change Organization Name**.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/manage-org-settings.md:35]:
```suggestion
     * Cloud Flex: At the top, select **Manage Organization > Change Organization Name**.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
done!

File: docs/manage/manage-subscription/manage-org-settings.md [hunk 3]
```diff
@@ -44,8 +44,8 @@ After you make this change, you will not be able to edit the account owner.
 
 1. In the left navigation bar of the UI, select **Administration** > **Account**.
 1. Access the appropriate menu for your [account type](#availability):
-     * Cloud Flex Credits: At the top select **Manage Organization > Change Account Owner**.
-     * Cloud Flex: From the kebob menu at the top select **Change Account Owner**.
+     * Cloud Flex Credits: From the kebob menu at the top select **Change Account Owner**.
+     * Cloud Flex: At the top select **Manage Organization > Change Account Owner**.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/manage-org-settings.md:48]:
```suggestion
     * Cloud Flex Credits: From the kebab menu at the top, select **Change Account Owner**.
     * Cloud Flex: At the top, select **Manage Organization > Change Account Owner**.
```

File: docs/manage/manage-subscription/manage-org-settings.md [hunk 4]
```diff
@@ -102,8 +102,8 @@ You must be the account owner of the Sumo Logic account to change the account su
 
 1. Go to **Administration** > **Account** and select the details icon at the top.
 1. Access the appropriate menu for your [account type](#availability):
-     * Cloud Flex Credits: At the top select **Manage Organization > Change Account Owner**.
-     * Cloud Flex: From the kebob menu at the top select **Change Account Subdomain**.
+     * Cloud Flex Credits: From the kebob menu at the top select **Change Account Subdomain**.
+     * Cloud Flex: At the top select **Manage Organization > Change Account Owner**.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/manage-org-settings.md:106]:
```suggestion
     * Cloud Flex Credits: From the kebab menu at the top, select **Change Account Subdomain**.
     * Cloud Flex: At the top, select **Manage Organization > Change Account Owner**.
```

**Comment by jpipkin1** [comment]
Thanks, @kimsauce.

**Comment by jpipkin1** [comment]
@kimsauce, could you approve this PR?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**